kf_lib_data_ingest/common/file_retriever.py

```diff
@@ -224,10 +280,26 @@ def get(self, url, auth=None):
                     dir=self.storage_dir,
                     delete=self.cleanup_at_exit and not self.__tmpdir
                 )
+
+                # Validate auth configs dict if it exists
+                # Select one auth config based inspection of url
+                auth_config = None
+                if self.auth_configs:
+                    self._validate_auth_configs(self.auth_configs)
```

Comment:
Following the above comment on preserving the ability to pass in an auth_config, you'd want to validate that too...
_validate_auth_configs could be changed to only validate a single auth config, validate the chosen auth config object down on line 291 under `if auth_config:` before the log statement, and then you could also loop the self.auth_configs items and validate its entries during init where it gets set.

Reply:
Ok, I removed that stuff bc I thought we didn't want that. I can put it back